docs/content/backup_restore.md

```diff
@@ -1,9 +1,129 @@
 # Backup and Restore
 
-WIP ...
+The *DbToolsBundle* comes with two Symfony Console commands to backup and restore
+your database but also a tiny backups manager (`DbToolsStorage`) which handle backup
+files for you.
 
 ## Backup command
 
+The backup command will use [defined binary]('./configuration#binaries') for your database vendor with correct parameters
+to dump your database.
+
+```sh
+console db-tools:backup
+```
+You can specify the behaviour of the command with some options:
+
+### Connection
+
+Per default, the command will backup the database from the default DBAL connection.
+
+You can choose to backup a database from another connection with `--connection` option:
+
+```sh
+console db-tools:backup --connection other_connection_name
+```
+
+### Excluded tables
+
+You may have configured [tables to be excluded in the bundle configuration](./configuration#excluded-tables).
+If so, these tables will be automatically excluded each time you launch the command.
+
+But if you want to temporary exclude some tables, run the command with the `--excluded-table` option:
+
+
+```sh
+# Exclude a table
+console db-tools:backup --excluded-table table_to_exclude
+
+# Or more
+console db-tools:backup --excluded-table table_to_exclude_1 --excluded-table table_to_exclude_2
+```
+
+### No cleanup
+
+Each time you launch the backup command, [backup file is store in a directory](./configuration#storage-directory).
```

Comment:
I probably would have documented the cleanup and its configuration before documenting how to disable it.